newsletter_laravel/app/Console/Commands/ExportNewsletterStories.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Services\NewsletterService;
+use Exception;
+use Illuminate\Console\Command;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use League\Csv\Writer;
+
+class ExportNewsletterStories extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'app:export-newsletter-stories';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Export all the stories with Newsletter review type';
+
+    public function __construct(NewsletterService $newsletterService)
```

Comment:
This parameter can be moved to `handle` method like `handle(NewsletterService $...)`. So you can completely remove constructor here.